addons/website/static/src/snippets/s_countdown/000.js

```diff
@@ -94,7 +94,7 @@ const CountdownWidget = publicWidget.Widget.extend({
      */
     _handleEndCountdownAction: function () {
         if (this.endAction === 'redirect') {
-            const redirectUrl = this.el.dataset.redirectUrl;
+            const redirectUrl = this.el.dataset.redirectUrl || '/';
```

Comment:
Could be an independant commit in old versions, but ok, probably not happening anyway.